problems/fizzbuzz.yaml

```diff
@@ -32,4 +32,4 @@ solution: |
       return num;
 
     return str;
-  }
+  };
```

Comment:
This semicolon is unnecessary and will actually trigger a warning from codemirror linter.